Readme.md

```diff
@@ -54,6 +55,17 @@ MusicBrainz for Android is a mobile app for viewing MusicBrainz data on the Andr
 ## Contributing
 	  
 Got **something interesting** you'd like to **ask or share**? Start a discussion at `#metabrainz` IRC channel on libera.chat.
+
+## Roadmap
+
+Proposed future plans for the app are as follows:
+
+- Introduce Jetpack Compose to the app.
+- Support more entities for search.
+- Ability to add entities natively from the app.
+- Showcase ListenBrainz functionalities to discover a wide range of users based on musical similarities.
```

Comment:
We should rephrase this to say, consider and add other *Brainz powered features.

Reply:
Sounds good!